zips/zip-0233.md

```diff
@@ -19,224 +20,103 @@ The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED", "OPTIONAL",
 and "REQUIRED" in this document are to be interpreted as described in RFC 2119.
 [1]
 
-The term "network upgrade" in this document is to be interpreted as described
-in ZIP 200. [2]
+The term "network upgrade" in this document is to be interpreted as described in
+ZIP 200. [2]
 
-"Block Subsidy” - the algorithmic issuance of ZEC on block creation – part of
-the consensus rules. Split between the miner and the Dev Fund. Also known as
-Block Reward.
+"Block Subsidy” - the algorithmic issuance of ZEC on block creation. Part of the
+consensus rules. Split between the miner and the Dev Fund. Also known as Block
+Reward.
 
-"Issuance" - The method by which unmined or unissued ZEC is converted to ZEC
-available to users of the network
+"Issuance" - The method by which ZEC becomes available for circulation on the
+network.
 
-"We" - the ZIP authors, owners listed in the above front matter
+"We" - the ZIP Owners and Authors, listed in the above front matter.
 
-"`MAX_MONEY`" is the ZEC supply cap. For simplicity, this ZIP defines it to be
-`21,000,000 ZEC`, although this is slightly larger than the actual supply cap
-of the original ZEC issuance mechanism.
+"`MAX_MONEY`" is the total ZEC supply cap, defined as 21,000,000 ZEC. This is
+slightly larger than the supply cap for the current issuance mechanism, but is
+the value used in existing critical consensus checks.
 
-# Abstract 
+# Abstract
 
-This ZIP describes the motivation, the necessary changes for, and the
-implementation specifications for the Zcash Sustainability Fund (ZSF). The ZSF
-is a proposed alteration to the block rewards system and accounting of unmined
-ZEC that allows for other sources of funding besides unissued ZEC. This new
-mechanism for deposits -- that new applications or protocol designs can use to
-strengthen the long-term sustainability of the network -- will likely be an
-important step for future economic upgrades, such as a transition to Proof of
-Stake or Zcash Shielded Assets, and other potential protocol fees and user
-applications.
-
-The changes in this ZIP are ultimately minimal, only requiring for the node to
-track state in the form of a `ZSF_BALANCE`, and for a new transaction field to
-be added, called `ZSF_DEPOSIT`. While wallet developer would be encouraged to
-add the `ZSF_DEPOSIT` field to their UIs, no changes or new behavior are
-absolutely required for developers or ZEC holders.
-
-This ZIP does not change the current ZEC block subsidy issuance schedule. Any
-additional amounts paid into the sustainability fund are reserved for use in
-future ZIPs.
+We propose the introduction of a mechanism to voluntarily burn funds, removing
+those funds entirely from circulation on the network. This mechanism, in
+combination with ZIPs 234 and 235, comprises a long-term strategy for the
+sustainability of the network. We will refer to the combined effects of these
+three ZIPs as the “Network Sustainability Mechanism”.
```

Comment:
@nuttycom, @str4d, and I think that this may not be a well-justified change in terminology, because:
* in other cryptocurrencies like Bitcoin, burnt funds can *never* be reissued;
* the term "burning" may conjure up associations that are not accurate in this context.

"Unissuance" seems fine to us.

Reply:
> in other cryptocurrencies like Bitcoin, burnt funds can never be reissued

But that _is_ also the intention here. It seems there could be legal complications with the idea of funds being somehow held in trust or otherwise kept and re-issued. So the intention is to permanently destroy the funds, and allow the economy to react. If this ZIP were taken in isolation, the only effect would be deflation, similarly to Ethereum. But in the context of ZIP-234, which redefines the issuance curve in terms of the chain value, the effect of reducing the chain value takes on additional significance.